src/bundle/DependencyInjection/IbexaTestCoreExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Test\Core\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+final class IbexaTestCoreExtension extends Extension
+{
+    /**
+     * @param array<string, mixed> $configs
+     */
```

Comment:
Well, if I'm already looking at this I'm gonna nitpick :sweat_smile: 
```suggestion
     *
     * @throws \Exception
     */
```